.travis.yml

```diff
@@ -1,3 +1,4 @@
+dist: trusty
```

Comment:
I updated to ubuntu 14.04 here to resolve a problem compiling native extensions for commonmarker. See: https://travis-ci.org/mcfiredrill/forttree/builds/235902726#L355